index.html

```diff
@@ -455,6 +459,15 @@ <h2>Legal Effects</h2>
             object to direct marketing under legitimate interest ([[?GDPR]]).
           </p>
         </li>
+        <li>
+        <p>
+           The DPA's goal is to "strengthen the control and personal autonomy of data subjects over their
```

Comment:
I think it would help if each of these paragraphs included a direct identification for the jurisdiction.  That is "Mauritius DPA" and "European Union GDPR" and "California CCPA".  That's not your fault necessarily, but an existing problem.

Reply:
@martinthomson I think that's a good point. PR incoming to resolve this separately.